ops/cmd/import_devnet/main.go

```diff
@@ -0,0 +1,137 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/ethereum-optimism/optimism/op-deployer/pkg/deployer/inspect"
+	"github.com/ethereum-optimism/optimism/op-deployer/pkg/deployer/state"
+	"github.com/ethereum-optimism/superchain-registry/ops/internal/config"
+	"github.com/ethereum-optimism/superchain-registry/ops/internal/manage"
+	"github.com/ethereum-optimism/superchain-registry/ops/internal/output"
+	"github.com/ethereum-optimism/superchain-registry/ops/internal/paths"
+	"github.com/urfave/cli/v2"
+)
+
+var (
+	StateFilename = &cli.StringFlag{
+		Name:      "state-filename",
+		Usage:     "Filename of an op-deployer state file.",
+		Required:  true,
+		TakesFile: true,
+	}
+)
+var (
+	ManifestPath = &cli.StringFlag{
+		Name:      "manifest-path",
+		Usage:     "Path to a manifest.yaml file specifying chain names.",
```

Comment:
Might be nice to be able to provide either this file _or_ the superchain name and a comma separated list of individual chain names.

Reply:
On the other hand, for any devnet being added without a manifest file on hand, could the user just create one with the minimal fields? Might actually be easier than threading everything through the CLI interface.